### PR TITLE
[enc2] Fixing JS errors

### LINKED ISSUE #15707: [enc2] JS error on personal page
encryption.js?v=93bf3ef7a60ae31d74d71d44ce8088a8:39 
Uncaught TypeError: undefined is not a function

### LINKED ISSUE #15760: TypeError: OC.Encryption.displayEncryptionWarning is not a function
Printed to the console with encryption enabled and when your private key needs to be re-passworded:
```
TypeError: OC.Encryption.displayEncryptionWarning is not a function
```

You can see that before Step5 of https://github.com/owncloud/core/pull/15677

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Fixes #15707 
Fixes #15760 

@th3fallen @schiesbn

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@th3fallen @schiesbn @nickvergessen @PVince81 please review - thx

**Comment by nickvergessen** [comment]
Still works and fixes the problem with the "not a function" :+1:

**Comment by DeepDiver1975** [comment]
@MorrisJobke please review - thx

File: apps/encryption/js/encryption.js [hunk 1]
```diff
@@ -5,37 +5,27 @@
...
-/**
- * @namespace
- * @memberOf OC
- */
-OC.Encryption= {
-	MIGRATION_OPEN: 0,
-	MIGRATION_COMPLETED: 1,
-	MIGRATION_IN_PROGRESS: -1,
 
+function displayEncryptionWarning () {
```
Comment (PVince81) [inline apps/encryption/js/encryption.js]: Why remove the namespace and pollute the global namespace ?

If the problem was that OC.Encryption wasn't defined at this stage, you can add this at the top:
```javascript
if (!OC.Encryption) {
   OC.Encryption = {};
}

```

Comment (DeepDiver1975) [inline apps/encryption/js/encryption.js]: the problem is that OC.Encryption is already defined - no clue how to fix that.

*Did I even mention that I hate JavaScript?*

Comment (PVince81) [inline apps/encryption/js/encryption.js]: If it is already defined, then it should work already. The `if` block I suggested is there to only create the namespace if it doesn't exist. We also do this in other places, see https://github.com/owncloud/core/blob/master/apps/files/js/app.js#L17

Comment (PVince81) [inline apps/encryption/js/encryption.js]: *Did I mention that I also used to hate Javascript ? Now it's only meh* :smile:

**Comment by DeepDiver1975** [comment]
@PVince81 please review - THX

File: apps/encryption/js/encryption.js [hunk 2]
```diff
@@ -5,25 +5,19 @@
  * See the COPYING-README file.
  */
 
-/**
- * @namespace
- * @memberOf OC
- */
```
Comment (PVince81) [inline apps/encryption/js/encryption.js]: Why remove this ? :see_no_evil:

File: apps/encryption/js/settings-personal.js [hunk 3]
```diff
@@ -4,20 +4,26 @@
...
+OC.Encryption = {
+	updatePrivateKeyPassword: function() {
+		var oldPrivateKeyPassword = $('input:password[id="oldPrivateKeyPassword"]').val();
+		var newPrivateKeyPassword = $('input:password[id="newPrivateKeyPassword"]').val();
+		OC.msg.startSaving('#encryption .msg');
+		$.post(
+			OC.generateUrl('/apps/encryption/ajax/updatePrivateKeyPassword'),
+			{oldPassword: oldPrivateKeyPassword, newPassword: newPrivateKeyPassword}
+		).success(function (response) {
+				OC.msg.finishedSuccess('#encryption .msg', response.message);
```
Comment (nickvergessen) [inline apps/encryption/js/settings-personal.js]: intendation is off?

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
Works and fixes both issues :+1:

**Comment by PVince81** [comment]
also :+1: for the comment changes + namespace

**Pull request merged**